### PR TITLE
Fix arguments to use the 'append' mode [CLOUDDST-501]

### PR DESCRIPTION
**Pull request by querti**

### CODE REVIEW

File: tests/test_iib_ops.py [hunk 1]
```diff
@@ -357,6 +357,48 @@ def test_add_bundles_py(
...
+    fixture_pulp_client.return_value.search_repository.return_value = [repo]
+    fixture_pulp_client.return_value.get_repository.return_value = repo
+    with setup_entry_point_py(
+        ("pubtools_iib", "console_scripts", "pubtools-iib-add-bundles"),
+        {"PULP_PASSWORD": "pulp-password", "CNR_TOKEN": "cnr_token"},
+    ) as entry_func:
+        retval = entry_func(
+            ["cmd"]
+            + fixture_common_iib_op_args
+            + ["--bundle", "bundle1", "--bundle", "bundle2"]
```
Comment (nathanegillett) [inline tests/test_iib_ops.py:380]: Although it seems to accept both values, the JIRA description says the command "should be `--bundle bundle1 bundle2`", as the help text for the arg describes it as a "comma separated list".

Comment (querti) [reply tests/test_iib_ops.py:380]: There is an inconsistency in how pub constructs pubtools-iib arguments vs how pubtools-iib accepts them. Pub constructs them as '--bundle bundle1 --bundle bundle2' while pubtools-iib accepts them as '--bundle bundle1 bundle2'. This causes a bug where passing '--bundle bundle1 --bundle bundle2' makes pubtools-iib accept only the second one.

My original solution was to modify pub to create the arguments in a way that pubtools-iib accepts them, but Jindrich said that I should actually change the way pubtools-iib accepts them, so here we are.

The comma separated values refers to how pub accepts this argument, so it doesn't have anything to do with this PR.

Comment (nathanegillett) [reply tests/test_iib_ops.py:380]: Ah, okay. The JIRA refers to the iib command, so maybe that should be updated.

Comment (chandwanitulsi) [reply tests/test_iib_ops.py:380]: +1 to updating JIRA

**Pull request merged**